client/checkout/classic/upe.js

```diff
@@ -145,11 +146,39 @@ jQuery( function ( $ ) {
 		$( '#wcpay_selected_upe_payment_type' ).val( paymentType );
 	};
 
+	// Set the selected UPE payment type field
```

Comment:
Super minor - I think this needs to be `Get`
```suggestion
	// Get the selected UPE payment type field
```